CITATION.cff

```diff
@@ -0,0 +1,186 @@
+    website: http://www.axonlab.org
+    orcid: https://orcid.org/0000-0001-8435-6191
+    affiliation:
+      Department of Radiology, Lausanne University Hospital and University of Lausanne,
+      Lausanne, Switzerland
+    email: phd@oscaresteban.es
+
+  - family-names: Gau
+    given-names: Rémi
+    affiliation: Université catholique de Louvain
+    email: remi.gau@uclouvain.be
+    orcid: "https://orcid.org/0000-0002-1535-9767"
+
+  - family-names: Halchenko
+    given-names: Yaroslav
+    affiliation: Dartmouth College
+    website: https://centerforopenneuroscience.org/whoweare#yaroslav_o_halchenko_
+    orcid: "https://orcid.org/0000-0003-3456-2493"
+
+  - family-names: Johnson
+    given-names: Timothy
+    affiliation: University of Michigan, Biostatistics
+    email: tdjtdj@umich.edu
+    website: https://sph.umich.edu/faculty-profiles/johnson-timothy.html
+
+  - family-names: Kent
+    given-names: James
+    affiliation:
+      Psychoinformatics Laboratory at the University of Texas at Austin
+    website: https://jdkent.github.io/
+    email: jamesdkent21@gmail.com
+
+  - family-names: Maumet
+    given-names: Camille
+    website: https://camillemaumet.com/
+    orcid: https://orcid.org/0000-0002-6290-553X
+    affiliation: Inria, Univ Rennes, CNRS, Inserm, Rennes, 35042, France
+
+  - family-names: Mumford
+    given-names: Jeanette
+    website: https://github.com/jmumford
+    email: jeanette.mumford@gmail.com
+
+  - family-names: Narayan
+    given-names: Manjari
+    email: manjari@alumni.rice.edu
+
+  - family-names: Nichols
+    given-names: Thomas E.
+    website: http://nisox.org
+    orcid: https://orcid.org/0000-0002-4516-5103
+    affiliation: University of Oxford, Big Data Institute, Oxford, OX3 7LF, UK
+    email: thomas.nichols@BDI.OX.AC.UK
+
+  - family-names: Nielson
+    given-names: Dylan
+    affiliation:
+      Machine Learning Team at the National Institute of Mental Health
+    website: https://github.com/Shotgunosine
+
+  - family-names: Niso
+    given-names: Guiomar
+    website: http://guiomarniso.com
+    orcid: https://orcid.org/0000-0001-5872-8924
+    affiliation: Instituto Cajal, CSIC, Spain
+    email: guiomar.niso@ctb.upm.es
+
+  - family-names: Ombao
+    given-names: Hernando
+    affiliation: KAUST, Saudi Arabia
+    website: https://cemse.kaust.edu.sa/biostats
+    email: hernando.ombao@kaust.edu.sa
+    orcid: https://orcid.org/0000-0001-7020-8091
+
+  - family-names: Poldrack
+    given-names: Russell A.
+    website: http://www.poldracklab.org
+    orcid: https://orcid.org/0000-0001-6755-0259
+    affiliation: Stanford University, Stanford, CA 94305 USA
+    email: russpold@stanford.edu
+
+  - family-names: Poline
+    given-names: Jean-Baptiste
+    website: https://github.com/jbpoline
+    orcid: https://orcid.org/0000-0002-9794-749X
+
+  - family-names: Thirion
+    given-names: Bertrand
+    website: https://github.com/bthirion
+    orcid: https://orcid.org/0000-0001-5018-7895
+
```

Comment:
`  
- family-names: Gorgolewski
    given-names: Krzysztof Jacek
    website: https://chrisgorgolewski.org`

Reply:
Added in f6604b2b404f0580af32ff9342292bbfbc64294f.